src/data/api/Questionnaire.ts

```diff
@@ -1,14 +1,61 @@
+import { Question } from '../../models/Question';
 import { Questionnaire } from '../../models/Questionnaire';
 import { apiEndpoints } from '../routes.json';
 import * as ApiRequest from './ApiRequest';
+import { generateFormDataByQuestion } from '../../helpers/formDataHelper';
 
-async function getQuestionnaireById(id: String): Promise<Questionnaire> {
-  const response = await ApiRequest.getRequest(
+async function getQuestionnairesByCodes(codes: string[]): Promise<Questionnaire[]> {
+  const response = await ApiRequest.postRequest(
```

Comment:
getQuestionnairesByCodes is een postRequest?

Reply:
Jup. Axios kan in een get request geen body meegeven blijkbaar. En deze endpoint verwacht een body dus is nu een post geworden